fixed_frame: "locator"
human_frame: "radio_position"
use_3d: false
mount_height: 0.0